st/clitests/awss3api.py

```diff
@@ -196,6 +196,21 @@ def list_objects_prefix_delimiter(self, bucket_name, max_keys=None, prefix=None,
         self.with_cli(cmd)
         return self
 
+    def list_object_versions(self, bucket_name, kwargs_options=None):
+        self.bucket_name = bucket_name
+        extra_args = ""
+        if kwargs_options is not None:
+            for arg in ("prefix", "delimiter", "page-size", "starting-token", "max-items", "encoding"):
```

Comment:
can we use "encoding-type" in arg list so that we can get rid of extra if/else condition.

Reply:
Had tried that, but was getting error with use of "-type" hence did not use.

Reply:
ok